pkg/events/events.go

```diff
@@ -75,6 +75,7 @@ type Event struct {
 	time      time.Time
 	eventtype EventType
 	values    map[string]string
+	object    interface{}
```

Comment:
we had not considered passing an object in the event previously - but it looks logical